### PR TITLE
newer results

### PR DESCRIPTION
**Pull request by doutriaux1**
@gleckler1 @durack1 @dnadeau4 this is a bit puzzling

These are the diff using esmpy. Denis fixed a problem with masking in this version but it's dreadfully slow (52mn for the test suite to run). What worries me is that the results are very different. We need to take a careful look. Please do NOT merge this.

### CODE REVIEW

**Comment by doutriaux1** [comment]
I do want you guys to take a look and comment please!

**Comment by doutriaux1** [comment]
The pmp code is identical only cdms2 changed.

**Comment by gleckler1** [comment]
Lets meet Tuesday to discuss how we compare the versions and quantify %wise how the results differ.    We need to understand the discrepancies.

**Comment by dnadeau4** [comment]
is it still slow with the latest way to mask data in CDMS?

**Comment by dnadeau4** [comment]
We could go on webex today,  I am here until 6:30pm.

**Comment by doutriaux1** [comment]
The good news is that Denis' new fast version give similar/identical results as the slow one. I think we are good and that it was due to the esmf misinterpolation.

File: test/pcmdi/customRegions/tas_2.5x2.5_regrid2_linear_metrics.json [hunk 1]
```diff
@@ -806,6 +806,23 @@
...
+        "platform": {
+            "OS": "Linux",
+            "Version": "4.8.0-41-generic",
+            "Name": "drdoom"
+        },
+        "userId": "doutriaux1",
+        "osAccess": false,
+        "commandLine": "/home/doutriaux1/anaconda2/envs/test_nightly/bin/pcmdi_metrics_driver.py -p test/pcmdi/region_specs_test.py",
+        "date": "2017-03-31 12:05:11",
+        "conda": {},
```
Comment (durack1) [inline test/pcmdi/customRegions/tas_2.5x2.5_regrid2_linear_metrics.json]: @doutriaux1 looks like this provenance stuff is no propagating..

**Comment by doutriaux1** [comment]
@durack1 the provenance is not the issue here, it's ignored by the terst suite, the issue is the number difference

**Comment by durack1** [comment]
@doutriaux1 got you, it could be useful to generate some diff maps to try and understand this issue.. At the same time it could be a good idea to determine the `OS X` vs `linux` differences #303 - they may be related

**Comment by gleckler1** [comment]
@doutriaux1  Not up to speed on this one.   Lets talk about it.  Generating difference maps definitely a good idea.

**Pull request closed without merging**